pkg/plugin/backup_pvc_action_plugin.go

```diff
@@ -100,15 +100,25 @@ func (p *NewPVCBackupItemAction) Execute(item runtime.Unstructured, backup *vele
 		return nil, nil, errors.WithStack(err)
 	}
 
+	// Do nothing if restic is used to backup this PV
+	isResticUsed, err := pluginUtil.IsPVCBackedUpByRestic(pvc.Namespace, pvc.Name, kubeClient.CoreV1(), boolptr.IsSetToTrue(backup.Spec.DefaultVolumesToRestic))
+	if err != nil {
+		return nil, nil, errors.WithStack(err)
+	}
+	if isResticUsed {
+		p.Log.Infof("Skipping PVC %s/%s, PV %s will be backed up using restic", pvc.Namespace, pvc.Name, pv.Name)
+		return item, nil, nil
+	}
+
 	if utils.IsFeatureEnabled(kubeClient, constants.CSIMigratedVolumeSupportFlag, false, p.Log) {
 		if pv.Spec.PersistentVolumeSource.CSI == nil && !pluginUtil.IsMigratedCSIVolume(pv) {
 			p.Log.Infof("Skipping PVC %s/%s, associated PV %s is not a CSI volume, nor a migrated volume", pvc.Namespace, pvc.Name, pv.Name)
-			return item, nil, nil
+			return nil, nil, errors.New("Only vsphere CSI or vsphere CSI migrated volume is supported")
```

Comment:
I think we should print out PVC namespace, name and PV name in the error message as well. So I suggest format the message (combine message on line 115 and 116) first, then print it out in Infof, and use the same formatted message when returning the error.

Also the first letter of a returned error message should be lower case.